app/views/users/show.html.erb

```diff
@@ -26,17 +26,17 @@
         <div class="card-header tab-card-header">
           <ul class="nav nav-tabs card-header-tabs" id="myTab" role="tablist">
             <li class="nav-item">
-                <a class="nav-link" id="one-tab" data-toggle="tab" href="#one" role="tab" aria-controls="One" aria-selected="true">Post</a>
+              <a class="nav-link" id="one-tab" data-toggle="tab" href="#one" role="tab" aria-controls="One" aria-selected="true">Post</a>
```

Comment:
You should use link_to instead of a tag if you can

Reply:
`<%= link_to "#one", "id": "one-tab", {class: "nav-link", data: { "toggle" => "tab"}, "role": "tab", "aria-controls": "One", "aria-selected": true} do %> Post`

Reply:
I am not sure how to do it, can you teach me to do one?